src/tui/mode/input/vim/visual.zig

```diff
@@ -208,6 +208,7 @@ fn mapPress(self: *Self, keypress: u32, egc: u32, modifiers: u32) !void {
 
             'o' => self.seq(.{ "smart_insert_line_before", "enter_mode" }, command.fmt(.{"vim/insert"})),
 
+            '`' => self.cmd("switch_case", .{}),
```

Comment:
Why would this not be identical to normal mode?

Reply:
thats how it works in vim, in visual mode it just switches your selection and moves your cursor back to the beginning, no advancing the cursor
<https://asciinema.org/a/aIgiFI9X6ojghM0J1nzubjP4J>

Reply:
ok